about.html

```diff
@@ -0,0 +1,148 @@
+                    <li><a href="#join">Join</a></li>
+                    <li><a href="#sponsor">Sponsor</a></li>
+                    <li><a href="#news">News</a></li>
+                </ul>
+                <button type="submit" class="campaign">Learn more</button>
+            </div>
+
+            <!-- desktop version  -->
+        <div class="list_navs">
+            <ul>
+                <li><a href="./about.html">About</a></li>
+                <li><a href="#project">Project</a></li>
+                <li><a href="#join">Join</a></li>
+                <li><a href="#sponsor">Sponsor</a></li>
+                <li><a href="#news">News</a></li>
+            </ul>
+            <button type="submit" class="campaign">Learn more</button>
+        </div>
+            
+        </nav>
+         <!-- head content  -->
+        <div class="head_content">
+            <h2>Find that <br>elusive Unicorns</h2>
+        </div>
+        <div class="header_quote">
+            <p>We at Sano have been in business since 2018, we have assist over millions of people find the right unicorn to invest into . We discover Unicorns with potential of becoming a successful startups and help them look for potential investors and also help investors by giving them the good investment opportunity.</p>
+        </div>
+        <div class="invitation">
+            <p>Please contact us through email for futher </p>
+            <p>questions about SANO</p>
+            <p class="underline"><strong>support@sano.org</strong></p>
+        </div>
+    </div>
+     <!-- about section  -->
+    <div class="logo" id>    
+        <div class="logo_content">
+            <h4>ABOUT SANO LOGO</h4>
+            <img src="./images/line2.png" alt="line" width="160px" height="20px">
+            <p class="logop"><strong>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Soluta molestias nam similique numquam amet magni,<br> 
+                voluptates facere consequuntur<br> 
+                fugit aliquid illo incidunt quod accusantium
+                 laudantium nesciunt. Itaque, quis. Temporibus, 
+                 ad!
+            </strong></p>
+                 <img class="aboutlogo" src="./images/thelogo (1).png" alt="logo" width="250px" height="250px">
+        </div>    
+    </div>
+
+    <!-- past section  -->
+    <div class="past_content">
+
+        <ul class="gallery">
+            <li class="group">
+                <img src="./images/group1.jpg" alt="group1">
+              <div class="group-info">
+                <h2 class="past-year">2017</h2>
+                <p>SANO sumit in Lagos 2017</p>
+              </div>
+            </li>
+            <li class="group">
+                <img src="./images/group2.jpg" alt="group2">
+                <div class="group-info">
+                    <h2 class="past-year">2018</h2>
+                    <p>SANO sumit in Abuja 2018</p>
+                  </div>
+            </li>
+          </ul>
+
+    </div>
```

Comment:
- [ ] The background of this section is white. kindly consider changing the background image to a white background, so your page looks exactly as the provided design

|Your Page|CC design|
|-|-|
|![Screenshot_20221123_202543](https://user-images.githubusercontent.com/25227022/203631234-d072da1d-5a16-4cdc-9aa8-8db3938ecd9b.png)|![Screenshot_20221123_202859](https://user-images.githubusercontent.com/25227022/203631252-4e469381-9948-4ae7-a32c-49bef466f1d5.png)|